tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/GradleFileChecker.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.upgrade.problems.core.internal;
+
+import com.liferay.ide.core.Artifact;
+import com.liferay.ide.core.util.FileUtil;
+import com.liferay.ide.gradle.core.parser.GradleDependencyUpdater;
+import com.liferay.ide.upgrade.problems.core.FileSearchResult;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.io.FileUtils;
+
+/**
+ * @author Seiphon Wang
+ */
+public class GradleFileChecker {
```

Comment:
Hey @Seiphon  can you move all of the methods in this class into the `GradleFileMigrator` instead?  thanks